### DISCUSSION TITLE
최대 에이전트의 개수는 몇 개인가요? What is the maximum number of agents?

### DISCUSSION THREAD

**Discussion by KANG-JI-HUN** [naver/ngrinder#873] [Q&A] [answered]
![캡처1](https://user-images.githubusercontent.com/48777577/165477531-5bb9a713-2bcf-41bf-a172-35c6c6ee5579.PNG)
![캡처2](https://user-images.githubusercontent.com/48777577/165477537-7de054f4-37f1-424d-b187-c7cb7102b694.PNG)
![캡처3](https://user-images.githubusercontent.com/48777577/165477538-1d2f27b1-1e54-4a56-b656-e04b6e4f2393.PNG)

에이전트의 수를 20개로 세팅해놓은 상태인데,
'테스트 생성' 후 테스트를 진행해보려고 했을 때
에이전트의 최대 개수가 10개로 밖에 표출되지 않습니다.
혹시 에이전트의 개수를 20개로 테스트해 볼 수 있는 방법이 있을까요?

**Comment by junoyoon** [comment, marked as answer]
시스템 설정에 
controller.max_agent_per_test=10
이라고 되어 있는 부분을 원하시는 값으로 수정하면 됩니다.

**Reply by KANG-JI-HUN** [reply]
감사합니다!
